lib/item.rb

```diff
@@ -0,0 +1,8 @@
+class Item
+  attr_reader :name, :price
+  def initialize(attributes = {})
```

Comment:
I would only set a default value if I expect that an Item will be created with no arguments.  Since the interaction pattern always sends a hash, we probably don't need the default value here.